modules/ROOT/pages/afm-run-test-now.adoc

```diff
@@ -5,19 +5,30 @@ endif::[]
 
 :imagesdir: ../assets/images
 
-You can run a test on demand without waiting for the current interval of its schedule to elapse. Doing so does not reset the schedule, which continues to run at previously set intervals.
+You can run a test before it is next scheduled to run. Doing so does not reset the schedule, which continues to run at previously set intervals.
 
 == Before You Begin
 
-Because you are running the test outside of a schedule, you must select the location from which to run it. If you want to run it in the same location as that used by the schedule, you can click the *Schedules* tab to look at the schedule and see at a glance the location that it uses.
+Because you are running the test outside of a schedule, you must select the location for the test to run. If you want to use the same location that is used by the schedule, navigate to the monitor detail page and view the location of the schedule.
```

Comment:
```suggestion
Because you are running the test outside of a schedule, you must select the location in which you want it to run. If you want to use the same location as used by the schedule, navigate to the monitor detail page and note that location for future use.
```